packages/sign-client/test/concurrency/concurrency.spec.ts

```diff
@@ -123,8 +123,8 @@ describe("Sign Client Concurrency", () => {
               async (resolve) => {
                 const timeout = setTimeout(() => {
                   log(`Client ${i} hung up`);
-                  resolve({ handshakeLatencyMs: 0, pairingLatencyMs: 0 });
-                }, 90_000);
+                  resolve({ handshakeLatencyMs: -1, pairingLatencyMs: -1 });
```

Comment:
The only comment I'd have is to make these values constants i.e.  what does `-1` mean here?

Reply:
I now I slipped this past you - will update later to something more meaningful as we keep iterating on the load test :)